static/src/stylesheets/module/facia/_mixins.scss

```diff
@@ -22,15 +22,15 @@ $breakpoints: (
             margin-left: $gs-gutter;
             margin-right: $gs-gutter;
         }
-        @include mq(containerWidestMobile) {
```

Comment:
Cool! No sense changing it now, but for future probably best to move this to a separate PR so the change can be properly documented in isolation (I have a vague memory of us discussing it, but if I didn't have that memory there would be no documented reason why this change happened)